orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -183,9 +188,6 @@ class OperationsController {
       pipeline = preprocessor.process(pipeline)
     }
 
-    def json = objectMapper.writeValueAsString(pipeline)
-    log.info('received pipeline {}:{}', value("pipelineId", pipeline.id), json)
```

Comment:
This logging seems to be just noise.

Reply:
Are you "replacing" this with the logs that either are 'started' or 'failed'?